en/docs/next/ai-workspace/llm-providers/configure-provider.md

```
@@ -155,5 +165,5 @@ The **Deployed Gateways** section lists all gateways this provider is deployed t
 
 ## Next Steps
 
-- [Configure App LLM Proxy](../llm-proxies/configure-proxy.md) - Configure and deploy a specialized proxy endpoint for a Gen AI application or agent using your provider
-- [Manage Provider](manage-provider.md) - Configure access control, security, rate limiting, and more
\ No newline at end of file
+- [Configure an App LLM proxy](../llm-proxies/configure-proxy.md): configure and deploy a specialized proxy endpoint for a GenAI application or agent that uses your provider
+- [Manage an LLM provider](manage-provider.md): configure access control, security, rate limiting, and more
```

markdownlint-cli2 (0.23.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)